content/WebAuthn/WebAuthn_Walk-Through.adoc

```diff
@@ -0,0 +1,283 @@
+== WebAuthn Walk-Through
+Build an app that enables your users to register and authenticate through WebAuthn. To implement the WebAuthn security protocol in your app, use the link:https://www.w3.org/TR/webauthn/[W3C Web Authentication API] specification and Yubico’s link:https://github.com/Yubico/java-webauthn-server/tree/master/webauthn-server-demo[Java WebAuthn Server].
+
+=== Overview
+With WebAuthn (also known as FIDO2), public-key cryptography is used to authenticate users to an online service, also known as a Relying Party (RP). When the end-user registers for an online service, an RP-specific credential key pair, i.e., a private key and a public key, is generated on the authenticator and the public key is sent to the RP (the private key never leaves the authenticator). When the user makes the request to log in, the authenticator sends an assertion that proves the user possesses the private key. The RP uses the public key to validate the assertion before allowing the user to log in.
+
+With WebAuthn also comes the concept of a resident key credential stored on the authenticator instead of encrypted and stored on the server. Resident keys enable passwordless authentication scenarios.
+
+For a detailed description of the registration and authentication ceremonies, see the “How it works” section of the link:WebAuthn_Developer_Guide/Overview.adoc[Overview] in the WebAuthn Developer Guide. In the same chapter, the sections immediately following, “FIDO2/WebAuthn application architecture” and “Key points” are also helpful.
+
+
```

Comment:
I would explicitly state here that we are going to walk through the Java-WebAuthn-Server's registration and authentication ceremonies and explain how the WebAuthn interaction works step-by-step.

Reply:
We subsequently agreed that this text can stay as it is.